readme.md

```
@@ -1,6 +1,9 @@
-# Universal Apps | ci.dev.br
+<img src="https://apps.ci.dev.br/logo-colorida.svg">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)